### ISSUE TITLE
Convert hard coded routes to the routes name 

### ISSUE THREAD

**Issue by pankaj-ag** [ColoredCow/employee-portal#588]
Currently, we have some hardcoded routes in the blade files. We need to convert them in route names.

**Comment by pankaj-ag** [comment]
@lalitmehta933, yes you need to check all the files in the resources/views directory and replace all the hard code routes with route name.

like in resources/views/finance/menu.blade.php you can replace  **/clients**  to **route('clients.index')** 

let me know if you have any other query on this.

**Comment by abhishek-pokhriyal** [comment]
@rathorevaibhav this is fixed. what remains is being tracked in #657
